### PR TITLE
SWV-10609 - Replace Tile Extraction Script

### PR DESCRIPTION
**Pull request by geprzemyslawwagner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,8 +101,15 @@ The configuration file can contain multiple areas to request tiles for by adding
 
 The parameters in the file are as follows:
 * **description** - A string containing a description of the area
+* **serverprotocol** - The protocol to be used for server connection ("http" or "https").
```
Comment (gedanieldutoit) [inline README.md]: Could you please make the serverprotocol optional and default it to https?  Or should it default to http for backwards compatibility?

**Review by gedanieldutoit — CHANGES_REQUESTED** [review]
Hi @geprzemyslawwagner 

Looks good to me.  Just one comment.

File: README.md (hunk 1 above)
Comment (geprzemyslawwagner) [reply README.md]: Done. It now defaults to 'http' so old config files should work without changing them.

**Review by gedanieldutoit — APPROVED** [review]
Hi @geprzemyslawwagner 

Looks good to me.

**Pull request merged**